src/lib/builders/combobox/create.ts

```diff
@@ -231,6 +235,16 @@ export function createCombobox<Value>(props?: CreateComboboxProps<Value>) {
 		return (item: Value) => deepEqual($value?.value, item);
 	});
 
+	/**
+	 * Determines if a given item is highlighted.
+	 * This is useful for displaying additional markup on the highlighted item.
+	 */
+	const isHighlighted = derived([highlightedItem], ([$value]) => {
+		return (item: Value) => {
+			return deepEqual($value ? getOptionProps($value).value : null, item);
```

Comment:
Couldn't you just use `highlighted` here instead of `highlightedItem`?

Reply:
yep, I built isHighlighted first lol. Good call